elasticsearch/templates/statefulset.yaml

```
@@ -136,6 +136,11 @@ spec:
           configMap:
             name: {{ template "elasticsearch.uname" . }}-jvm-options
         {{- end }}
+        {{- if .Values.createCert }}
+        - name: elasticsearch-certs
+          secret:
+           secretName: {{ template "elasticsearch.name" . }}-certs
```

https://devops-ci.elastic.co/job/elastic+helm-charts+pull-request+integration-elasticsearch/1249/ES_SUITE=default,KUBERNETES_VERSION=1.19,label=docker&&virtual/console is failing with `MountVolume.SetUp failed for volume "elasticsearch-certs" : secret "elasticsearch-certs" not found` error, because the secret cert is now named `elasticsearch-master-certs`.



```suggestion
           secretName: {{ template "elasticsearch.uname" . }}-certs
```